### ISSUE TITLE
How do I configure jscpd linter to ignore files?

### ISSUE THREAD

**Issue by jamiet-msm** [github/super-linter#1968]
I have a markdown document that jscpd (which highlights duplicated code) is complaining about. The duplications are valid though (because I am explaining various facets of the code and as a consequence I have different code blocks in my markdown that resemble each other).

I would like to configure jscpd to ignore my markdown file so I places a file `.jscpd.json` at the root of my repo containing:

```json
{
    "threshold": 0,
    "reporters": ["html", "console", "badge"],
    "ignore": ["path/to/README.md"],
    "absolute": true
  }
```
but still I'm getting errors when I run super-linter

```default
2021-09-19 12:09:06 [ERROR]   Found errors in [jscpd] linter!
2021-09-19 12:09:06 [ERROR]   Error code: 1. Command output:
------
Clone found (markdown):
 - /github/workspace/dataplatform/tests/features/README.md [168:1 - 189:15] (21 lines, 168 tokens)
   /github/workspace/dataplatform/tests/features/README.md [129:1 - 150:15]

Clone found (markdown):
 - /github/workspace/dataplatform/tests/features/README.md [245:1 - 261:17] (16 lines, 109 tokens)
   /github/workspace/dataplatform/tests/features/README.md [209:1 - 225:12]

Clone found (markdown):
 - /github/workspace/dataplatform/tests/features/README.md [279:1 - 297:7] (18 lines, 114 tokens)
   /github/workspace/dataplatform/tests/features/README.md [209:1 - 263:7]

Clone found (markdown):
 - /github/workspace/dataplatform/tests/features/README.md [313:1 - 331:43] (18 lines, 112 tokens)
   /github/workspace/dataplatform/tests/features/README.md [209:1 - 263:43]
```

I've tried changing various things in the `ignore` attribute in `.jscpd.json` but none seem to have an affect. 

Can anyone tell me how I can tell jscpd to ignore certain files?

**Comment by tunetheweb** [comment]
Put your config file in .github/linters folder is normal process.

Looks like there is a base one in the TEMPLATES folder that will be used if you don’t override it with your own as per above: https://github.com/github/super-linter/blob/master/TEMPLATES/.jscpd.json

**Comment by jamiet-msm** [comment]
That did it, thx @tunetheweb
